ISeriesProvider/DB2iSeriesSqlBuilder.cs

```diff
@@ -13,14 +13,21 @@ namespace LinqToDB.DataProvider.DB2iSeries
 
 	public class DB2iSeriesSqlBuilder : BasicSqlBuilder

```

Comment:
Indentation changes got this messed up sorry about that. The actual changes here are options in constructor and their usage, and the rename of iSeriesDummyTableName to GetDB2DummyTableName.

The original constructor (ISqlOptimizer sqlOptimizer, SqlProviderFlags sqlProviderFlags,	ValueToSqlConverter valueToSqlConverter) is mandatory because Linq2db tries to resolve it with expression